### PR TITLE
chore: fix function names in comment

### PR DESCRIPTION
**Pull request by HongKuang**

### CODE REVIEW

**Review by sago35 — APPROVED** [review]
LGTM

**Comment by deadprogram** [comment]
Thanks for the corrections @HongKuang and to @sago35 for review. Now merging.

**Comment by deadprogram** [comment]
@HongKuang also please note that I changed the branch for this PR to `dev` before merging.

See https://tinygo.org/docs/guides/contributing/#how-to-use-our-github-repository for more information.

**Comment by HongKuang** [comment]
> @HongKuang also please note that I changed the branch for this PR to `dev` before merging.
> 
> See https://tinygo.org/docs/guides/contributing/#how-to-use-our-github-repository for more information.

Thanks！

File: src/os/tempfile.go [hunk 1]
```diff
@@ -142,7 +142,7 @@ func joinPath(dir, name string) string {
 	return dir + string(PathSeparator) + name
 }
 
-// LastIndexByte from the strings package.
```
Comment (dkegel-fastly) [inline src/os/tempfile.go:145]: oh, booger.  This typo fix broke the meaning of the comment.  Might want to amend this to give the new function name and then the old function name if you are motivated enough.

**Pull request merged**